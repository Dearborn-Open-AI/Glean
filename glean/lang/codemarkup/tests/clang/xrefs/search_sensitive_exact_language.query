query: codemarkup.search.SearchByName { searchcase = Sensitive, name = "foo", kind = { just = Function }, language = Cpp }
perf: true
transform: [gensort, []]
