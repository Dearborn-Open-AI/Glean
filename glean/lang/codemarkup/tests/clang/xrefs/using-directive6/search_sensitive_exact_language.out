[
  "@generated",
  {
    "key": {
      "searchcase": 0,
      "name": "foo",
      "entity": {
        "cxx": {
          "decl": {
            "function_": {
              "key": {
                "name": {
                  "key": {
                    "name": { "key": { "name": { "key": "foo" } } },
                    "scope": {
                      "namespace_": { "key": { "name": { "key": "N1" } } }
                    }
                  }
                },
                "signature": {
                  "key": { "returns": { "key": "void" }, "parameters": [ ] }
                },
                "source": {
                  "file": { "key": "test.cpp" },
                  "lineBegin": 19,
                  "columnBegin": 1,
                  "lineEnd": 21,
                  "columnEnd": 1
                }
              }
            }
          }
        }
      },
      "location": {
        "name": "foo",
        "file": { "key": "test.cpp" },
        "location": {
          "range": {
            "file": { "key": "test.cpp" },
            "lineBegin": 19,
            "columnBegin": 1,
            "lineEnd": 21,
            "columnEnd": 1
          }
        },
        "span": { "start": 296, "length": 3 }
      },
      "kind": 13,
      "language": 0
    }
  }
]